python-threatexchange/threatexchange/signal_type/tests/test_hash_from_x.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import unittest
+import os
+
+from threatexchange.content_type import meta
+from threatexchange.signal_type.signal_base import FileHasher, StrHasher
+from pathlib import Path
+
+
+class TestHashFrom(unittest.TestCase):
+    """
+    Signal Type has a weird inheritance structure the error around which are easy to
+    miss before run time. If a SignalType is a FileHasher/StrHasher it should be able to
+    take the corresponding content medium without throwing.
```

Comment:
Update this comment - it's not an inheritance issue, just a pair of bugs:
* Refactor missed a method rename
* copy-paste fail from match to hash cmd accidentally attempting to call load(), which is not needed for the command